docs/pages/versions/unversioned/sdk/background-task.mdx

```diff
@@ -179,6 +181,79 @@ Since the Background Tasks API on iOS and the WorkManager API on Android limit t
 
 The last registered background task determines the minimum interval for execution.
 
+## Testing Background Tasks
+
+Background tasks can be tested using the [`triggerTaskWorkerForTestingAsync`](#backgroundtasktriggertaskworkerfortestingasync) method. This method will invoke the `BGTaskScheduler` on iOS and run all registered tasks directly on Android. This is useful for testing the behavior of your background tasks without having to wait for the system to trigger them.
+
+This method is only available in development mode. It will not work in production builds.
+
+```tsx
+import * as BackgroundTask from 'expo-background-task';
+import { Button } from 'react-native';
+
+function App() {
+  const triggerTask = async () => {
+    await BackgroundTask.triggerTaskWorkerForTestingAsync();
+  };
+
+  return <Button title="Trigger Background Task" onPress={triggerTask} />;
+}
+```
+
+## Inspecting background tasks&ensp;<PlatformTags platforms={['android']} />
+
+To troubleshoot or debug issues with background tasks on Android, use the `adb` tool included with the Android SDK to inspect scheduled tasks:
+
+```bash
```

Comment:
For commands, let's use Terminal component here and the bash snippets below: https://github.com/expo/expo/blob/main/docs/README.md#use-terminal-component-for-shell-commands-snippets